src/App.vue

```diff
@@ -753,31 +753,31 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
         switch (this.getFilingType) {
           case FilingTypes.AMALGAMATION_APPLICATION:
             if (this.isAmalgamationFilingRegular) {
-              this.$router.push(RouteNames.AMALG_REG_INFORMATION).catch(() => {})
+              this.$router.replace(RouteNames.AMALG_REG_INFORMATION).catch(() => {})
             } else if (this.isAmalgamationFilingHorizontal || this.isAmalgamationFilingVertical) {
-              this.$router.push(RouteNames.AMALG_SHORT_INFORMATION).catch(() => {})
+              this.$router.replace(RouteNames.AMALG_SHORT_INFORMATION).catch(() => {})
```

Comment:
This is so that the new route replaces the old one, without pushing the old one (which was invalid for the current filing type) to the browser history. It's a small optimization. It means that, if the user clicks the Back button, they won't navigate to the invalid route.